test/sync/w3c_web_element.dart

```diff
@@ -127,16 +128,22 @@ void runTests(config.createTestDriver createTestDriver) {
     test('findElement -- success', () {
       var element = table.findElement(const By.tagName('tr'));
       expect(element, config.isSyncWebElement);
-    });
+    });*/
 
     test('findElement -- failure', () {
       try {
         button.findElement(const By.tagName('tr'));
         throw 'Expected Exception';
-      } on Exception {}
+      } catch (e) {
+        expect(e, new isInstanceOf<W3cWebDriverException>());
+        expect((e as W3cWebDriverException).httpStatusCode, 404);
+        expect((e as W3cWebDriverException).error, 'no such element');
+        expect((e as W3cWebDriverException).message,
+            contains('Unable to locate element'));
+      }
     });
 
-    test('findElements -- 1 found', () {
+    /*test('findElements -- 1 found', () {
```

Comment:
Sorry, I've not figured out to to filter tests in package test by name. There is a method using tags but I find this faster in practice.

That said, I'm seeing a benefit to using tags. :-) (Good catch.)